main.go

```diff
@@ -262,10 +262,21 @@ func (s *slackOutput) sendMessage(msg slackMessage) error {
 	return fmt.Errorf("Retry limit `%d` exceeded posting slack message: %+v. Error: %s", s.retryLimit, msg, lastError)
 }
 
+func (s *slackOutput) hasNotifications(rawmsg []byte) bool {
+	var msg = string(rawmsg)
+
+	return (strings.Contains(msg, "_kvmeta") && strings.Contains(msg, "notifications")) ||
+		strings.Contains(msg, "oom-killer") || strings.Contains(msg, "notification_alert_type")
```

Comment:
consider using constants (e.g. `var kvmetaBytes = []byte("_kvmeta")`) + `bytes.Contains(rawmsg, ...)` for max speed